### PR TITLE
Feature ETP-791: Add host parameter and code unification

### PR DESCRIPTION
**Pull request by valenvivaldi**

### CODE REVIEW

File: src/com/etendoerp/openapi/model/init/InitialClientSetupEndpoint.java [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+        initialClientSetupRequestSchema,
+        initialClientSetupRequestExample,
+        initialClientSetupResponseSchema,
+        initialClientSetupResponseExample,
+        commonHeaders, tags,
+        "POST");
+  }
+
+  /**
+   * Método para crear el endpoint relacionado con Initial Client Setup
```
Comment (JaviBert) [inline src/com/etendoerp/openapi/model/init/InitialClientSetupEndpoint.java]: Docs must be in english

File: src/com/etendoerp/openapi/model/init/InitialOrganizationSetupEndpoint.java [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+        initialOrgSetupRequestSchema,
+        initialOrgSetupRequestExample,
+        initialOrgSetupResponseSchema,
+        initialOrgSetupResponseExample,
+        commonHeaders, tags,
+        "POST");
+  }
+
+  /**
+   * Método para crear el endpoint relacionado con Initial Org Setup
```
Comment (JaviBert) [inline src/com/etendoerp/openapi/model/init/InitialOrganizationSetupEndpoint.java]: Docs must be in english

File: src/com/etendoerp/openapi/model/printreport/PrintDocumentEndpoint.java [hunk 3]
```diff
@@ -22,8 +22,18 @@
 public class PrintDocumentEndpoint implements OpenAPIEndpoint {
 
   private static final String PRINT_DOCUMENT_ACTION = "com.etendoerp.client.print.PrintOptionsActionHandler";
-  private static final String BASE_PATH = "/etendo/orders/sws/PrintOptions.html";
+  private static final String BASE_PATH = "/etendo/orders/PrintOptions.html?stateless=true";
```
Comment (JaviBert) [inline src/com/etendoerp/openapi/model/printreport/PrintDocumentEndpoint.java]: Is correct delete sws?

File: src/com/etendoerp/openapi/model/window/WindowSettingEndpoint.java [hunk 4]
```diff
@@ -24,6 +26,15 @@ public class WindowSettingEndpoint implements OpenAPIEndpoint {
   private static final String WINDOW_SETTINGS_ACTION = "org.openbravo.client.application.WindowSettingsActionHandler";
   private static final String FORM_INIT_ACTION = "org.openbravo.client.application.window.FormInitializationComponent";
   private static final String BASE_PATH = "/etendo/org.openbravo.client.kernel?_action=";
+  private static final List<String> tags = List.of("Window Settings");
+
+  @Override
+  public boolean isValid(String tag) {
```
Comment (JaviBert) [inline src/com/etendoerp/openapi/model/window/WindowSettingEndpoint.java]: This method is in three different java files. Wouldn't it be better for this to be in a "Util" file?

**Review by JaviBert — CHANGES_REQUESTED** [review]
Some docs are in spanish

File: src/com/etendoerp/openapi/model/window/WindowSettingEndpoint.java (hunk 4 above)
Comment (valenvivaldi) [reply src/com/etendoerp/openapi/model/window/WindowSettingEndpoint.java]: It uses a private propertie so its not possible. Additionally this method can be different for each class that implements the Interface

File: src/com/etendoerp/openapi/model/printreport/PrintDocumentEndpoint.java (hunk 3 above)
Comment (valenvivaldi) [reply src/com/etendoerp/openapi/model/printreport/PrintDocumentEndpoint.java]: Yes because uses another auth way

**Comment by valenvivaldi** [comment]
i will rebase the development to avoid the conflicts

**Review by fran-roig — APPROVED** [review]

**Pull request merged**